packages/@angular/cli/bin/ng

```diff
@@ -98,8 +98,9 @@ if (packageJsonProjectPath && fs.existsSync(packageJsonProjectPath)) {
   const hasDep = !!deps['@angular/cli'];
   const hasOldDevDep = !!devDeps['angular-cli'];
   const hasDevDep = !!devDeps['@angular/cli'];
+  const hasDependencyWarning = CliConfig.fromProject().get('warnings.cliDependency');
```

Comment:
`fromGlobal()`, you don't have a project in `ng new`.

Reply:
Thanks for the review! Updated and squashed.

Reply:
Hmm, looks like I'll want to read from the project configuration if it is within a project. I'll make that change.

Reply:
With and without pulling in the project config:
![image](https://user-images.githubusercontent.com/4655972/36544096-95195484-17b3-11e8-9787-3051ad5d5db0.png)

Looks like `fromProject()` returns `null` when it's not available so I just have a conditional to swap between the two.